### ISSUE TITLE
Using MultilabelSupportVectorLearning<Gaussian>, meaning of parameter C?

### ISSUE THREAD

**Issue by wup11111** [accord-net/framework#1600]
**What would you like to submit?** (put an 'x' inside the bracket that applies)

  - [x] question
  - [ ] bug report
  - [ ] feature request

**Issue description**

Hi Cesar,

I use the particle swarm optimization algorithm to find the best classifier parameters C and σ on a particular data set. The evaluation criteria are the results of 100 random cross-validations using the MCMC sampling method to obtain training sets and test sets with different K values. I set the range of the parameters is 0-100. When the optimal classification result is found, the parameter C is approximately 55.23 and the parameter σ is approximately 5.84. I know that as the parameter C increases, the classifier will have a greater probability of overfitting, is the parameter C too large? Can you explain the meaning of the parameter C in the classifier and the range of common intervals? I also want to know how the classifier works if you don't input the parameter C. The parameter C is always the initial default value (1.0), or it will automatically find an optimal value? If the value of parameter C is not set and UseComplexityHeuristic is set to true, what method does the framework use to automatically guess the value of parameter C? What is the accuracy of this method? Do you think it is better to set UseComplexityHeuristic to be true, or is it better to use C as a variable to participate in the particle swarm algorithm? What other excellent methods do you have to find the best parameters for the classifier?
I hope you can reply to me, thank you very much!!!

example:
` var teacher_2_train = new MultilabelSupportVectorLearning()
                        {
                            / / Configure the learning algorithm to use SMO to train
                            // The underlying SVM in each binary class subproblem.
                            Learner = (param) => new SequentialMinimalOptimization()
                            {
                                / / Estimate the appropriate guess of the Gaussian kernel parameters.

                                // This estimate can be used as a starting point for grid search. The default value is true.
                                UseKernelEstimation = true,

                                //1. Set a complexity parameter C of a support vector machine
                                // The larger the C value, the less likely the classifier is to allow classification errors ("outliers").
                                // If the C value is too large, the classifier will do its utmost to make fewer mistakes in the training data, but in fact it is impossible/meaningless, thus causing overfitting.
                                //When the C value is too small, the classifier will be too "don't care" about the classification error, so the classification performance will be poor.
                                Complexity = X[num_c][0],

                                //2. Set the Gaussian radial basis kernel function parameter σ
                                Kernel = new Gaussian(X[num_c][1]),

                                //3. Complete the standard convergence tolerance of the model training process. The default value is 0.01.
                                Tolerance = 0.0001,

                            }
                        };`

Best regards.
Wu Peng
